specs/lint-complexity-reduction/request-log-segmentation.spec.md

```
@@ -0,0 +1,344 @@
+    this.providerMetrics.set(key, perf);
+  }
+  
+  getSummary(provider: string, account: string, model: string): PerformanceMetrics | undefined {
+    return this.providerMetrics.get(`${provider}:${account}:${model}`);
+  }
+}
+```
+
+## File Structure After Refactoring
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)